### PR TITLE
source-sense-data contribution from Martel-R

### PR DESCRIPTION
**Pull request by Martel-R**
Auto-generated PR by the Connector Builder for source-sense-data

### CODE REVIEW

**Comment by natikgadzhi** [comment]
Hey @Martel-R! Can you pretty please upload a square centered logo in icon.svg into this PR?

**Comment by Martel-R** [comment]
hi @natikgadzhi , I uploaded it, need anything else?

**Comment by Martel-R** [comment]
Thanks guys

File: airbyte-integrations/connectors/source-sense-data/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
+version: 4.6.2
+
+type: DeclarativeSource
+
+description: Conector de leitura dos dados Sense Data
```
Comment (natikgadzhi) [inline airbyte-integrations/connectors/source-sense-data/manifest.yaml:5]: Connector docs are not translated to different languages ;-( probably best to start with a sentence in English, but ok to keep this right after.

**Review by natikgadzhi — CHANGES_REQUESTED** [review]
I am not entirely against, but there is just one stream (clients list) and even that does not have pagination setup. 

This feels like a very poor feature set.

File: airbyte-integrations/connectors/source-sense-data/manifest.yaml (hunk 1 above)
Comment (Martel-R) [reply airbyte-integrations/connectors/source-sense-data/manifest.yaml:5]: Ok, added to backlog

**Comment by Martel-R** [comment]
I was learning how to use it, but I have already improved a lot in the usability of the platform.

**Comment by marcosmarxm** [comment]
@Martel-R, the connector looks great, but it currently has low coverage of streams and features (pagination and incremental updates). You haven’t signed the CLA, so we can’t merge your contribution. For now, I’m closing this, but you’re welcome to submit again later.

**Pull request closed without merging**